### PR TITLE
Change background form hypertension value field to free text

### PR DESCRIPTION
**Pull request by mcglk**
Per Claudia on 2021-05-27, this should be just a text field "so that
the user can put in a complete blood pressure reading
(systolic/diastolic) or a single number." Removed the "numeric
decimalformat" from the class, and removed the relevant validation
lines.

Also commented out the TSV generator in build.sh for the moment; I'm
working on a new one in any case as part of the audit.

### CODE REVIEW

File: docs/src/build.sh [hunk 1]
```diff
@@ -39,4 +39,4 @@ DOCS="${DIR}/doc"
 rm -rf "${DOCS}"
 jsdoc ${DIR}/../www/*.js --readme ${DIR}/../src/README.md -d "${DOCS}"
 
-python3 "${DIR}/tsv-dd-to-html.py"
\ No newline at end of file
+# python3 "${DIR}/tsv-dd-to-html.py"
```
Comment (djdenv) [inline docs/src/build.sh:42]: This change should not be committed as it bypasses the friendly HTML DD.

**Review by djdenv — CHANGES_REQUESTED** [review]
Please revert the change to the build script. Since the DD didn't change, it's ok that you can't regenerate it.

File: docs/src/build.sh (hunk 1 above)
Comment (mcglk) [reply docs/src/build.sh]: The script call has been restored in build.sh.

**Review by djdenv — APPROVED** [review]
Looks good. Please use the squash and merge option when merging.

**Review by djdenv — APPROVED** [review]

**Pull request merged**